client/src/app/site/motions/components/motion-detail/motion-detail.component.ts

```diff
@@ -1170,6 +1218,87 @@ export class MotionDetailComponent extends BaseViewComponent implements OnInit {
         this.repo.followRecommendation(this.motion);
     }
 
+    /**
+     * Function to send a notification, so that other persons can recognize editing the same motion, if they're doing.
+     *
+     * @param type TypeOfNotificationViewMotion defines the type of the notification which is sent.
+     * @param user Optional userId. If set the function will send a notification to the given userId.
+     */
+    private sendEditNotification(type: TypeOfNotificationViewMotion, user?: number): void {
+        const content: ViewMotionNotificationEditMotion = {
+            motionId: this.motion.id,
+            senderId: this.randomEditorId,
```

Comment:
In addition to this id, get the current username from the operator and include the short name in this notify message. The random token may not be necessary, becuase the server appends a `replayChannel` to every send message that uniqly identifies the client. Kepp a list of reply channels and user names to nicly format the message: `Users <A>, <B>, and 3 more are editing this motion`.

Reply:
done